vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -74,6 +74,16 @@ def _all_hidden(components: Component | list[Component]):
     },
 )
 
+_OuterPageDivsType = TypedDict(
```

Comment:
This one is also no longer just `Div` so should be renamed. As with my comments in #1267 it feels a bit "inconsistent scale" to have `collapsible-icon-div` on the same level as `right-side`. Is there any way we can fix that? Not a big problem if not though.